server/src/main/java/org/elasticsearch/index/store/Store.java

```diff
@@ -1063,7 +1063,7 @@ public int size() {
         }
 
         /**
-         * returns the history uuid the store points at, or null if not existant.
+         * returns the history uuid the store points at, or null if not existent.
```

Comment:
I think `not existent` should be `nonexistent`.

Reply:
Thanks. I've fixed it.